app/services/contributors_generator.rb

```diff
@@ -32,8 +32,14 @@ def initialize(work:)
   # H2 Publisher becomes a Cocina::Models::Event, not a Contributor.  See events_from_publisher_contributors.
   sig { returns(T::Array[T.nilable(Cocina::Models::Contributor)]) }
   def generate
+    count = 0
     work.contributors.reject { |c| c.role == 'Publisher' }
-        .map { |work_form_contributor| contributor(work_form_contributor) }
+        .map do |work_form_contributor|
+          count += 1
+          # First entered contributor is always status: "primary" (except for Publisher)
```

Comment:
@astridu @amyehodge  I'm not sure that the form makes it clear that the first author is "primary"

Reply:
@peetucket We may want to set up a brief check-in today about this. Also including @arcadiafalcone @ndushay @jcoyne and myself.

Reply:
We're setting up a meeting on Friday where we can discuss this.

Reply:
I'll convert this PR to draft, then.  I thought "primary" was a separate animal from "order" but perhaps not.